main.tf

```diff
@@ -0,0 +1,21 @@
+provider "aws" {
+     region  = "us-east-1"
+     access_key = "XXXX"
```

Comment:
Hardcoding access keys inside code is bad practice even if you remove them when committing to git.
There is always a chance that you accidentally commit your access key and secret key or forget to delete them from code.

Pelase obtain your keys from environment variables or from a credential file instead

Any of these options will work well:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables
https://registry.terraform.io/providers/hashicorp/aws/latest/docs#shared-configuration-and-credentials-files
https://registry.terraform.io/providers/hashicorp/aws/latest/docs#using-an-external-credentials-process